src/components/Slider/SliderFixedRange.tsx

```diff
@@ -0,0 +1,334 @@
+import React, {useCallback, useMemo} from 'react'
+import {StyleSheet} from 'react-native'
+import styled from 'styled-components/native'
+import {Thumb, Track, TrackPoint} from './components'
+import type {PanGestureHandlerGestureEvent} from 'react-native-gesture-handler'
+import type {AnimatedGHContext, AnimatedLabelProps, SliderCommonProps} from './Slider'
+import type {ITheme} from '../../theme'
+import {
+  useAnimatedStyle,
+  useSharedValue,
+  useAnimatedGestureHandler,
+  runOnJS,
+  useAnimatedProps,
+  withTiming,
+} from 'react-native-reanimated'
+import {
+  DEFAULT_MAXIMUM_VALUE,
+  DEFAULT_MINIMUM_VALUE,
+  DEFAULT_STEP,
+  DURATION,
+  FIRST_POINT,
+  INIT_POINT,
+  INIT_VALUE,
+  INVISIBLE,
+  MINIMUM_TRACK_WIDTH,
+  NEXT_STEP,
+  PREVIOUS_STEP,
+  ThumbPosition,
+  VISIBLE,
+} from './constants'
+import {useTheme} from '../../hooks'
+import {hitSlop} from '../../helpers/metrics'
+
+type Point = {
+  left: number
+  right: number
+}
+
+type SliderAnimated = {
+  opacity: number
+  zIndex: number
+}
+
+type ContainerProps = {
+  width: number
+}
+
+type NumberRange = {
+  maximum: number
+  minimum: number
+}
+
```

Comment:
export reuse in component SliderRange, not duplicate type again

Reply:
updated